x-pack/plugins/siem/public/mock/test_providers.tsx

```diff
@@ -58,3 +61,15 @@ export const TestProviders = pure<Props>(
     </I18nProvider>
   )
 );
+
+export const TestProviderWithoutDragAndDrop = pure<Props>(
+  ({
+    children,
+    store = createStore(state, apolloClientObservable),
+    mockFramework = mockFrameworks.default_UTC,
```

Comment:
`mockFramework` is an unused variable. Did you intend to pass it down? If not, you can remove it.